napi/README.md

```diff
@@ -0,0 +1,3 @@
+# markdown-rs
+
+CommonMark compliant markdown parser in Rust with ASTs and extensions. Packaged for npm with full WASM support.
```

Comment:
The readme would need to be fairly complete, a lot could probably be lifted from https://github.com/remarkjs/remark or https://github.com/micromark/micromark
Which this is based off

Reply:
Sure, I think this can be done in a follow-up PR